src/main/java/com/urbancode/jenkins/plugins/ucdeploy/VersionHelper.java

```diff
@@ -295,6 +295,8 @@ private void setComponentVersionProperties(
                 propSheetDef = compClient.getComponentVersionPropSheetDef(component);
                 propSheetDefId = (String) propSheetDef.get("id");
                 propSheetDefPath = (String) propSheetDef.get("path");
+                propSheetDefPath = propSheetDefPath.replaceAll("/", "&");
```

Comment:
Hi, 

I apologize, but this `/` -> `&` issue has not been able to be replicated locally. We can not accept this change without proper verification on our side. We have tried to replicate it using your version of UCD and Jenkins, but have been unsuccessful. From our viewpoint, the Jenkins plugin propSheetDef resolves as expected.

Have you been able to run the test code I sent through the escalation? This test should be able to isolate where the error is coming from. Please run the test and send the results.

-Nick